Sources/AirbnbSwiftFormatTool/airbnb.swiftformat

```diff
@@ -34,6 +34,7 @@
 --redundanttype inferred # redundantType, propertyType
 --typeblanklines preserve # blankLinesAtStartOfScope, blankLinesAtEndOfScope
 --emptybraces spaced # emptyBraces
+--operatorfunc no-space # spaceAroundOperators
```

Comment:
This has no effect for now, but will kick in after we enable the `spaceAroundOperators` rule in a follow-up